packages/location-management/src/components/LocationUnitAddEdit/Form.tsx

```diff
@@ -0,0 +1,287 @@
+import * as Yup from 'yup';
+import React from 'react';
+import { SubmitButton, Form as AntForm, Input, Radio, Select, TreeSelect } from 'formik-antd';
+import { Button } from 'antd';
+import { history } from '@onaio/connected-reducer-registry';
+import { getUser } from '@onaio/session-reducer';
+import { OpenSRPService } from '@opensrp/server-service';
+import { getAccessToken } from '@onaio/session-reducer';
+import { Formik } from 'formik';
+import {
+  LocationUnitPayloadPOST,
+  LocationUnitPayloadPUT,
+  LocationUnitStatus,
+  LocationUnitSyncStatus,
+  LocationUnitTag,
+} from '../../ducks/location-units';
+import { useSelector } from 'react-redux';
+import { Geometry } from 'geojson';
+import { LOCATION_HIERARCHY, LOCATION_UNIT_POST_PUT } from '../../constants';
+import { v4 } from 'uuid';
+import { LocationUnitGroup } from '../../ducks/location-unit-groups';
+import { ParsedHierarchyNode, RawOpenSRPHierarchy } from '../../ducks/types';
+import { sendErrorNotification, sendSuccessNotification } from '@opensrp/notifications';
+import { API_BASE_URL } from '../../configs/env';
```

Comment:
Let's retrieve the base URL from props

Reply:
@haseemVD have created a separate issue to deal with it .. it will be solved separately

Reply:
Created separate [issue](https://github.com/OpenSRP/web/issues/256) for this